app/assets/stylesheets/pages.css.scss

```diff
@@ -319,25 +319,51 @@ $include-html-global-classes: false;
   }
 }
 
-.rackspace {
+.footer {
   background-color: $hackedu-black;
-  color: $text-on-black;
-  padding: 10px 0 10px 0;
-
-  p {
-    font-size: rem-calc(14);
-    margin: 7.5px 0 7.5px 0;
+  color: $hackedu-white;
+  padding: 30px 0;
+  font-weight: 300;
 
-    @media #{$medium-up} {
+  .nav {
+    h1 {
+      text-transform: uppercase;
+      color: $hackedu-white;
       font-size: rem-calc(18);
-      margin: 15px 0 15px;
+      font-weight: 600;
+      margin-left: 5px;
     }
 
-    a {
-      color: scale-color($text-on-black, $lightness: 10%);
-      text-decoration: underline;
+    ul {
+      margin-left: 0;
+
+      li {
+        list-style-type: none;
+        padding: 5px;
+        color: $hackedu-white;
+        font-weight: inherit;
+
+        p {
+          font-weight: inherit;
+        }
+
+        a {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 5